infrahub_sdk/types.py

```diff
@@ -64,3 +66,7 @@ def exception(self, event: str | None = None, *args: Any, **kw: Any) -> Any:
 
 
 InfrahubLoggers = Union[InfrahubLogger, Logger]
+
+
+class OrderModel(BaseModel):
```

Comment:
I think we should call it just `Order` to keep it simpler for the user and easier to read